src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public class Lottery {
+    public void getRandomBall() {
+        Ball ball = new Ball();
+        ColorSupplier l = new ColorSupplier();
```

Comment:
no need to create an instance of `ColorSupplier` each time #getRandomBall is being invoked. 

think about creating the class level field of ColorSupplier and using it instead.